features/step_definitions/book_index_page_steps.rb

```diff
@@ -1,5 +1,7 @@
 Given(/^there are 100 books in the database$/) do
-  FactoryGirl.create_list(:book, 100)
+  FactoryGirl.create(:book, published_date: Date.today, order_count: 2)
+  FactoryGirl.create(:book, order_count: 1)
+  FactoryGirl.create_list(:book, 98)
 end
```

Comment:
One improvement I would make here is to take the `100` in as an argument, so if you had to do `Given there are 10 books in the database` you wouldn't have to write a new step.